### ISSUE TITLE
Suggestion: FAQ Blueprint to quickly add FAQ

### ISSUE THREAD

**Issue by chrisvidal** [artistro08/tailor-starter#85]

**Comment by artistro08** [comment]
I don't think this would benefit people who use the theme, considering how many people would use this feature. 

I'm also pretty sure you can create something like this with the current blocks (Without coding)

**Comment by chrisvidal** [comment]
I am pretty sure that 95% of websites are offering a FAQ. It's pretty standard nowadays and would make total sense in this starter.

**Comment by artistro08** [comment]
In that case, I'm willing to support a pull request for the blueprint, blocks, and pages.

**Comment by artistro08** [comment]
Since it's been a few weeks and no one has opened a pull request, I can handle this one if no one has anything in the works.

**Comment by artistro08** [comment]
Added in v6.2.24
